tests/models/test_licensing.py

```diff
@@ -1174,8 +1174,8 @@ def test_compatible_with_calls_compatible_with_on_deliverymechanism(self):
         # media types.
         edition, pool = self._edition(with_license_pool=True,
                                       with_open_access_download=True)
-        self._add_generic_delivery_mechanism(pool)
-        [mech1, mech2] = pool.delivery_mechanisms
+        [mech1] = pool.delivery_mechanisms
+        mech2 = self._add_generic_delivery_mechanism(pool)
```

Comment:
This test failed on Travis but not locally; I removed what could have been some nondeterministic code.